packages/rich-text/src/customExport.ts

```diff
@@ -40,13 +41,15 @@ function inlineStyleFn(styles) {
   };
 }
 
-function exportHTML(editorState) {
+function exportHTML(editorState: EditorState) {
   if (!editorState) {
     return null;
   }
 
   const html = stateToHTML(editorState.getCurrentContent(), {
     inlineStyleFn,
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment -- See the next line for reason.
+    // @ts-ignore Reason: `defaultBlockTag` is optional (undefined | string), however, `undefined` means `p` so we need to use `null` here.
     defaultBlockTag: null,
```

Comment:
Looks like there hasn't been a new release of `draft-js-export-html` [since 2019](https://www.npmjs.com/package/draft-js-export-html), so we can't really update it. But we can fork it, like I forked https://github.com/swissspidy/draft-js-import-element

Reply:
Damn, that's a long time ago. A fork sounds reasonable until we can move on to [facebook/lexical](https://github.com/facebook/lexical), when it goes to stable at some point in the hopefully _not-too-distant_ future.

Reply:
Hm, looks like the `inlineStyleFn` used above also gives a TS error, it doesn't seem to be among the accepted options at all 🤔 

Have to check why / how / if it works as expected currently.

Reply:
@barklund / @swissspidy If you have a moment, could one of you try if you use [this forked version of draft-js-export-html](https://github.com/miina/draft-js-export-html) locally if the `npm install` runs OK and the module is imported fine? 
It fails with deps errors for me and when running with `force`, the module is not resolved. Not sure if it's just a local problem or not. I saw the same error with the repo forked by Pascal before as well so maybe it's me 🤷

Reply:
The GitHub repository needs to contain the built files, not just the source files.

Briefly did that in https://github.com/swissspidy/draft-js-export-html and 54e12c6074332e01d28e1a1f5d6b62b15c454f18

Let's see how that goes.